SConstruct

```diff
@@ -1485,54 +1485,45 @@ def buildDocs( target, source, env ) :
 
 if haveSphinx and haveInkscape :
 
-	docEnv = commandEnv.Clone()
+	# We build docs in the standard environment rather than commandEnv, so we can
+	# use host python to avoid needing a matrix of sphinx versions to match Gaffer's
+	# bundled python version.
+	docEnv = env.Clone()
+	docEnv["ENV"]["PYTHONPATH"] = ":".join( sys.path )
```

Comment:
```suggestion
	docEnv["ENV"]["PYTHONPATH"] = env.subst( ":".join( sys.path + split( commandEnv["LOCATE_DEPENDENCY_PYTHONPATH"] ) ) )
```

Reply:
Awesome, thanks @andrewkaufman, I seem to remember it's a world of fun getting the sphinx + packages versions working together.

Reply:
@andrewkaufman Actually, @johnhaddon pointed out that `LOCATE_DEPENDENCY_PYTHONPATH` is meant to hold deps that are dependencies of `gaffer` itself, not the build process. If we include this in `docEnv`, we'll be re-coupling system python and gaffer python.

In a Gaffer 3 build, `LOCATE_DEPENDENCY_PYTHONPATH` would have python3 modules on it. However `scons` may still be in python 2. 

I've reverted the PR. Can you try building having added your python 2 compatible `sphinx` to the `PYTHONPATH` in the environment you're running `scons` from?

Reply:
> Can you try building having added your python 2 compatible sphinx to the PYTHONPATH in the environment you're running scons from?

Possibly simpler than that, you should also be able to modify `sys.path` directly from the IE config.

Reply:
Yep, that works for me:

```
diff --git a/config/ie/options b/config/ie/options
index 992c62c..ba1805e 100644
--- a/config/ie/options
+++ b/config/ie/options
@@ -311,8 +311,6 @@ LOCATE_DEPENDENCY_PYTHONPATH = [
 	os.path.join( IEEnv.Environment.rootPath(), "tools", "python", pythonVersion, IEEnv.platform(), compiler, compilerVersion, "subprocess32", "3" ),
 	os.path.join( IEEnv.Environment.rootPath(), "tools", "python", pythonVersion, IEEnv.platform(), compiler, compilerVersion, "PyOpenGL", "3" ),
 	os.path.join( IEEnv.Environment.rootPath(), "tools", "python", pythonVersion, "noarch" ),
-	os.path.join( sphinxRoot, "lib", "python" + pythonVersion, "site-packages" ),
-	os.path.join( sphinxRoot, "lib64", "python" + pythonVersion, "site-packages" ),
 	os.path.join( IEEnv.Environment.rootPath(), "apps", "Qt.py", qtPyVersion ),
 
 ]
@@ -417,6 +415,10 @@ CORTEX_PYTHON_LIB_SUFFIX = "-" + cortexMajorVersion + "-python" + pythonVersion
 
 os.environ["PATH"] = os.path.join( pythonReg["location"], compiler, compilerVersion, "bin" ) + ":" + os.environ["PATH"]
 os.environ["PYTHONHOME"] = os.path.join( pythonReg["location"], compiler, compilerVersion )
+sys.path.extend( [
+	os.path.join( sphinxRoot, "lib", "python" + pythonVersion, "site-packages" ),
+	os.path.join( sphinxRoot, "lib64", "python" + pythonVersion, "site-packages" ),
+] )
 ```

Reply:
Thanks @andrewkaufman I've added a commit [here](https://github.com/GafferHQ/gaffer/pull/3898/commits/711d7e1fb005b1bc79de07338787e35875bc40da) that updates the options for you.